.github/workflows/tests.yml

```diff
@@ -1,9 +1,12 @@
 name: Tests
 
 on:
-  pull_request:
-    branches: '*'
-
+  workflow_dispatch:
```

Comment:
Nice, you could keep both :)

Reply:
Thanks, I didn't know I could keep both!

Reply:
By the way, just adding the `workflow_dispatch` statement doesn't gonna change anything, unless you merge it with the default branch, which is master in this case. In order to make it work on my forked repo, I changed the default branch to `add-attributes-only`. I thought you mind find this useful.